### PR TITLE
Update existing "snapshot/{}/{}" endpoint with the new delegation field

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
Add new delegations field for the existing `snapshot/{}/{}` endpoint.
`snapshot/{}/{}` return value:
```
{"voting_power": voting_power, "voting_group": voting_group, "delegations": delegations}
```

### CODE REVIEW

**Review by saibatizoku — APPROVED** [review]

**Comment by Mr-Leshiy** [comment]
@minikin I have already added it :) or do you mean some specific place for it ?

**Comment by Mr-Leshiy** [comment]
firstly need to merge https://github.com/input-output-hk/catalyst-toolbox/pull/152

**Comment by minikin** [comment]
@Mr-Leshiy, I left a comment before you added `VoterInfo` 🤓

File: doc/api/v0.yaml [hunk 1]
```diff
@@ -844,25 +844,32 @@ components:
...
       type: array
       items:
-        $ref: "#/components/schemas/VotingPower"
+        $ref: "#/components/schemas/VotingInfo"
       example:
         [
           {
             "voting_power": 1000,
             "voting_group": "representative",
+            "delegations": 2,
```
Comment (Zeegomo) [inline doc/api/v0.yaml:856]: this is not among the fields in `VotingInfo`

File: doc/api/v0.yaml [hunk 2]
```diff
@@ -844,25 +844,32 @@ components:
...
 
-    VotingPower:
+    VotingInfo:
       properties:
         voting_power:
           type: integer
           format: u64
         voting_group:
           type: string
+        delegations_power:
```
Comment (Zeegomo) [inline doc/api/v0.yaml:867]: what is this used for?

**Review by Zeegomo — CHANGES_REQUESTED** [review]

**Pull request merged**